Microsoft.Toolkit.Uwp.UI.Controls/RangeSelector/RangeSelector.cs

```diff
@@ -247,7 +251,9 @@ private void Thumb_KeyUp(object sender, KeyRoutedEventArgs e)
                 case VirtualKey.Right:
                     if (_toolTip != null)
                     {
-                        _toolTip.Visibility = Visibility.Collapsed;
+                        dispatcherTimer.Debounce(
+                            () => _toolTip.Visibility = Visibility.Collapsed,
+                            TimeToHideToolTipOnKeyUp);
```

Comment:
We don't need need to make this an async call on the DispatchQueue because its using this method or do we?

Reply:
Good question @RosarioPulella, I didn't think of that initially. DispatcherTimer is supposed to remain on the UI thread, so it should handle that, and we don't need to await it, as we don't care about the result. However, I'm not sure if the `DispatcherTimer` is still using `Dispatcher` internally or not which could be a problem for the future...

@azchohfi was `DispatcherTimer` updated to use `DispatcherQueue` internally? Or is this resolved with the [WinUI version here](https://docs.microsoft.com/en-us/windows/winui/api/microsoft.ui.xaml.dispatchertimer) in the future? Are we safe to continue using this?

Reply:
No, it was not. We have the DispatcherQueueTimer now. You can call the dispatcherQueue.CreateTimer() to create a DispatcherQueueTimer associated with that DispatcherQueue.

Reply:
The `Debounce` extension only exists for the `DispatchTimer`...
I can try to make the same extension for the `DispatcherQueueTimer` not sure if it would go in `Microsoft.Toolkit.Uwp.UI` with the other `Debounce` method?

Reply:
@RosarioPulella let's make sure we do it as a move for the `DispatcherTimerExtensions` to `DispatcherQueueTimerExtensions`.